README.md

```diff
@@ -1,23 +1,112 @@
+# Sample - NOS3 Component
+This repository contains the NOS3 Sample Component.
+This includes flight software (FSW), ground software (GSW), simulation, and support directories.
 
-# Sample - NOS3 cFS Application
+## Overview
+This sample component is a UART device that accepts multiple commands, including requests for telemetry and data.
+The available FSW is for use in the core Flight System (cFS) while the GSW supports COSMOS.
+A NOS3 simulation is available which includes both sample and 42 data providers.
 
-This repository contains the NOS3 Sample cFS application.
 
-## Overview
-The example assumes a UART based device that.  This example can be used
-with the sample simulator to show a device that is either synchronous or
-asynchronous (i.e. can stream data).  The sample shows how to maintain
-and report application status, how to maintain and report device status,
-and how to automatically or on command send device data.  When showing
-device streaming, the streaming rate can be commanded.
-
-## Documentation
-Currently support exists for the following versions:
-* [cFS 6.7/6.8](https://cfs.gsfc.nasa.gov/)
-
-# Commanding
-Refer to the file `fsw/platform_inc/sample_app_msgids.h` for the Sample app message IDs
-Refer to the file `fsw/src/sample_app_msg.h` for the Sample app command codes
-
-## Versioning
+# Device Communications
+The protocol, commands, and telemetry of the component are captured below.
+
+## Protocol
+The protocol in use is UART 115200 8N1.
+The device is speak when spoken too.
+All communications with the device require / contain a header of 0xDEAD and a trailer of 0xBEEF.
+
+## Commands
+All commands received by the device are echoed back to the sender to confirm receipt.
+Should commmands involve a reply, the device immediately sends the reply after the command echo.
+Device commands are all formatted in the same manner and are fixed in size:
+* uint16, 0xDEAD
+* uint8, command identifier
+  - (0) Get Housekeeping
+  - (1) Get Sample
+  - (2) Set Configuration
+* uint32, command payload
+  - Unused for all but set configuration command
+* uint16, 0xBEEF
+
+## Telemetry
+Telemetry formats are as follows:
```

Comment:
https://github.com/nasa/nos3/pull/109#issuecomment-1353405028
So from the physical hardware accepts commands and produces responses. 
The FSW accepts commands and reports telemetry.
@msuder - Is that what you're thinking the terminology should be updated to reflect?

Reply:
Yes

Reply:
Resolved in commit 31ce518